tests/foreman/cli/test_variables.py

```diff
@@ -410,17 +647,23 @@ def test_positive_validate_default_value_with_list(self):
 
         @steps:
 
-        1.  Create a variable with some default value that matches the list of
-        step 2.
-        2.  Validate this value with list validator type and rule.
+        1.  Create a variable with default value that matches the values from
+        validator list
 
         @assert: Variable is created for matched validator rule.
-
-        @caseautomation: notautomated
         """
+        smart_variable = make_variable({
+            'variable': gen_string('alpha'),
+            'puppet-class': self.puppet['name'],
+            'default-value': 'test',
+            'validator-type': 'list',
+            'validator-rule': '5, test',
```

Comment:
please pass it as a list  - `'validator-rule': [5, 'test'],`, `Base._construct_command()` should deal with it.